src/presenter/point-presenter.js

```diff
@@ -72,10 +70,10 @@ export default class PointPresenter{
   }
 
   #renderViewMode() {
-    const view = this.#createViewModeComponent(this.#pointData);
-    this.#renderOrReplace(view);
+    this.#view = this.#createViewModeComponent(this.#pointData);
```

Comment:
Ты и так устанавливаешь переменную класса  `this.#pointViewComponent` зачем ещё одна `this.#view` ?
```suggestion
  const view = this.#createViewModeComponent(this.#pointData);
      this.#renderOrReplace(view);

    this.#pointViewComponent = view;
```